### ISSUE TITLE
Cleaned Classification Data with Validation & Test Splits

### ISSUE THREAD

**Issue by rhn19** [goru001/nlp-for-marathi#4]
The Marathi News Dataset contains only the Train and Validation Splits. This made it hard to report accuracy and compare to the baseline (which reports the validation accuracy). Hence, there's a need to have separate Train-Validation-Test sets.
Also, the headlines contain non-Devanagari characters (mostly English) and Arabic numerals. These need to be mapped to Devanagari characters and numerals. The original dataset contains ~50 duplicates which need to be cleaned.

Following the above steps, I have created new splits of the dataset ([link here](https://www.kaggle.com/rhn1903/marathi-news-dataset-cleaned)) which can be used for properly reporting scores on the Classification dataset.

**Comment by rhn19** [comment]
Apparently all of the datasets in iNLTK don't have a separate test set and the accuracy is reported on the validation set. The models provided are chosen based on this validation accuracy. This may cause inefficiency in model generalization. Opening this issue there.
